lib/actions/logs.coffee

```diff
@@ -42,14 +35,13 @@ module.exports =
 	]
 	permission: 'user'
 	action: (params, options, done) ->
-		resin.logs.subscribe params.uuid, {
+		resin.logs.subscribe(params.uuid, {
```

Comment:
The parentheses are not needed here. We tend to omit them for multi line function calls.